src/App.tsx

```diff
@@ -27,11 +28,24 @@ function getReorderedGoods(
   sortType: SortType,
   isReversed: boolean,
 ) {
-  // Not to mutate the original array
   const visibleGoods = [...goods];
 
-  // Sort and reverse goods if needed
-  // ...
+  visibleGoods.sort((prev, next) => {
```

Comment:
Variable names should be meaningful instead of `prev and next` you could use `product1 and product2` for example.